deploy/docs/v1_conf_examples.md

```diff
@@ -107,7 +117,7 @@ fluentd:
 
```

Comment:
In the S3 option above, should we also call out how to properly include the existing output pipieline? Since the conf file has helm template syntax they need to resolve that yeah?

Reply:
good point, will add that